README.md

```diff
@@ -1,4 +1,12 @@
 Disclaimer: suboptimal in many ways, but more than enough for what I need.
 
+## Command line args
+
+`bs` accepts the following command line arguments:
+
+`-p`: It checkouts to the selected branch and executes a `git pull`
+    immediately afterwards.
+`-f`: It executes a `git fetch` _before_ showing the branches to be selected. It's useful to update branches before diplay.
```

Comment:
`-f` usually stands for `--force`, I'd go with `-F`

Reply:
Or even `-u` for `update`. Whatever you prefer

Reply:
I see, very idiomatic of you! 👍